scripts/publish-plugin.sh

```diff
@@ -16,26 +19,70 @@ if [ "$(go env GOOS)" = "windows" ]; then
     BIN_SUFFIX=".exe"
 fi
 
-(cd "${ROOT}/provider" && go build \
-   -ldflags "-X github.com/pulumi/pulumi-kubernetes/provider/v2/pkg/version.Version=${VERSION}" \
-   -o "${WORK_PATH}/pulumi-resource-kubernetes${BIN_SUFFIX}" \
-   "${ROOT}/cmd/pulumi-resource-kubernetes")
+(cd provider && go build \
```

Comment:
I am not sure if the change in the line is ok. @metral

Reply:
For ref we do this in other PRs: 

- https://github.com/pulumi/pulumi-auth0/pull/2/files#diff-682a1507e4f9c0dae17cadd4bec21056L22
- https://github.com/pulumi/pulumi-aws/pull/973/files#diff-682a1507e4f9c0dae17cadd4bec21056R22

Reply:
Yes, I used `pulumi-digitalocean` for reference and it, too, was like `pulumi-auth0` from your link. I guess I should have clarified that I just wasn't sure if the way that this script is invoked in _this_ repo necessitates the `${ROOT}` prefix in the command. But it sounds like it is not so the change should be ok?

Reply:
Ah gotcha. IIUC `${ROOT}` does not seem to be needed @chrsmith ?